fe/fe-core/src/main/java/com/starrocks/catalog/CatalogRecycleBin.java

```diff
@@ -263,12 +264,15 @@ private synchronized void eraseTable(long currentTimeMs) {
                 // erase table
                 tableIter.remove();
                 idToRecycleTime.remove(tableId);
-
+                tableIdList.add(tableId);
                 // log
-                Catalog.getCurrentCatalog().getEditLog().logEraseTable(tableId);
-                LOG.info("erase table[{}-{}] finished", tableId, table.getName());
+                LOG.info("erase table[{}-{}] in memory finished", tableId, table.getName());
```

Comment:
Is it necessary? can we remove this log? Or print log outside the critical block?

Reply:
it is better to log which table is real  deleted, so we can know if it can be restored.
print log outside the critical block needs to put the table name in the Map and pass it to the outside.